src/cats-controller.js

```diff
@@ -526,6 +526,27 @@ function formatName(name) {
     .join(delimiter || '');
 }
 
+/**
+ * Удаление кота
+ * @param req
+ * @param res
+ */
+function removeCats(req, res) {
+  const { catId } = req.params
+
+  if (isEmpty(catId)) {
+    return res.status(400).json(boom.badRequest('cat id is absent'))
+  }
+
+  catsStorage.removeCats(catId)
+    .then(() => {
```

Comment:
Такого id может не быть
```suggestion
.then(removedCat => {
  if (!removedCat) {
    return res.status(404).json(boom.notFound('Не найден id кота'))
  }
```